lib/pure/parsecfg.nim

```diff
@@ -442,12 +442,12 @@ type
   Config* = OrderedTableRef[string, <//>OrderedTableRef[string, string]]
 
 proc newConfig*(): Config =
-  ## Create a new configuration table.
+  ## Creates a new configuration table.
   ## Useful when wanting to create a configuration file.
   result = newOrderedTable[string, <//>OrderedTableRef[string, string]]()
 
 proc loadConfig*(stream: Stream, filename: string = "[stream]"): <//>Config =
-  ## Load the specified configuration from stream into a new Config instance.
+  ## Loadw the specified configuration from stream into a new Config instance.
```

Comment:
@xflywind Loads

Reply:
Yeah, I will clean this in my fix PR.

Reply:
I sometimes batch miscellaneous small fixes into a combined PR but up to you however you want to do that